rasa/nlu/components.py

```diff
@@ -648,10 +654,15 @@ def can_handle_language(cls, language: Hashable) -> bool:
         """
 
         # if language_list is set to `None` it means: support all languages
-        if language is None or cls.language_list is None:
+        if language is None or (
+            cls.language_list is None and cls.not_supported_language_list is None
+        ):
             return True
 
-        return language in cls.language_list
+        language_list = cls.language_list or []
+        not_supported_language_list = cls.not_supported_language_list or []
+
+        return language in language_list or language not in not_supported_language_list
```

Comment:
Can we refactor `language_list` variable to `supported_language_list`? Took me a while to decipher what all `language` variables were there 😄